### ISSUE TITLE
The z-index of the WC top header is too high and a couple of strings are not translated

### ISSUE THREAD

**Issue by DAnn2012** [woothemes/woocommerce#37416]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

**1) The z-index of the WC top header is too high.**

It has a value of 1001 and this implies that the submenus of WP dashboard menu items are shown below the top header of WooCommerce.

**2) A couple of strings are not translated.** 

The strings "**days**" and "**Dismiss**" are shown in English even though the language of my WooCommerce installation is Italian.

### Expected behavior

The correct display of WP dashboard submenus and the two strings translated into Italian in the WooCommerce administration pages.

### Actual behavior

See in "**Describe the bug**"
![wc-header](https://user-images.githubusercontent.com/1197819/227466351-92e93d57-7a3e-4b1e-b7ee-c3552bdf7925.png)

### Steps to reproduce

1. Set the WP administration language to Italian.
2. Click on the menu item to view WooCommerce administration or analytics.
3. Click on a WordPress dashboard menu item with a submenu.

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): http://localhost/test
Site address (URL): http://localhost/test
WC Version: 7.5.1
REST API Version: ✔ 7.5.1
WC Blocks Version: ✔ 9.6.6
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.1.1
WP Multisite: ✔
WP Memory Limit: 384 MB
WP Debug Mode: –
WP Cron: ✔
Language: it_IT
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.3.33
PHP Post Max Size: 128 MB
PHP Time Limit: 60
PHP Max Input Vars: 5000
cURL Version: 7.51.0
OpenSSL/1.1.1k

SUHOSIN Installed: –
MySQL Version: 8.0.26
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Il tuo server non ha la classe SoapClient attiva - alcuni plugin gateway che usano SOAP potrebbero non funzionare correttamente.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Status report information ###

Generated at: 2023-03-24 09:22:49 +01:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by chihsuan** [comment]
Hi @DAnn2012 Thanks for reporting this!

> 1) The z-index of the WC top header is too high.

Unfortunately, I wasn't able to reproduce this. Could you share more details and confirm it's not caused by other plugins?

![Screenshot 2023-05-04 at 12 49 33](https://user-images.githubusercontent.com/4344253/236114931-3b25e662-952f-4009-834b-397c2d1784da.png)

>  The strings "days" and "Dismiss" are shown in English even though the language of my WooCommerce installation is Italian.

The string "days' is related to a [WP bug](https://core.trac.wordpress.org/ticket/47373). We'll fix "Dismiss" string soon!

**Comment by DAnn2012** [comment]
Hi @chihsuan 

Thank you for addressing my report.

Following your suggestion, I did some tests and I discovered that the problem does not occur only by deactivating the WP-ERP plugin from the developer weDevs that I had active on my test environment because I had received the assignment from one of my clients to translate it into Italian and I'm waiting for the developer to fix some issues with strings in the source code.

Forgive me for opening this report and not checking first if this was caused by other plugins and not WooCommerce.

However, I take this opportunity to ask you for the kindness to check if also on your WordPress and WooCommerce installation, activating this plugin causes the problem. So that I can report this discussion to the weDevs developer to warn him of the problem.

Thank You very much.
